Changes.md

```diff
@@ -26,6 +26,11 @@ Documentation
 
 - Node Reference : Removed duplicate entries for nodes that have been aliased by compatibility configs.
 
+Features
+--------
+
+- SelectionTool : Added selection modifier plug. When set to anything except `None` using the SelectionTool causes the actual scene location selected to potentially be modified from the originally selected location. Selection modifiers work identically for deselection. Currently, USD Kind modifiers are implemented. When selecting, the first ancestor location with a `usd:kind` attribute matching the chosen list of USD Kind will ultimately be selected. USD's Kind Registry includes `Assembly`, `Component`, `Group`, `Model` and `SubComponent` by default and can be extended via USD startup scripts.
```

Comment:
"selection modifier" -> "select mode". "None" -> "Any".

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/5720/commits/13cb3fbfe4bf47f4269ecb9b3598bc97b152aead, with "None" -> "Standard" to reflect that separate change.